packages/runner/src/app/app.jsx

```diff
@@ -152,11 +152,8 @@ class App extends Component {
       containerNode.className += ' screenshotting'
 
       if (!config.scale) {
-        const $window = $(window)
-        const $iframesSizeNode = $(iframesSizeNode)
-
-        iframesSizeNode.style.width = `${Math.min($window.width(), $iframesSizeNode.width())}px`
-        iframesSizeNode.style.height = `${Math.min($window.height(), $iframesSizeNode.height())}px`
+        iframesSizeNode.style.width = `${Math.min(window.innerWidth, iframesSizeNode.offsetWidth)}px`
```

Comment:
jQuery 3 changes how it does width and height calculations (it now uses `getBoundingClientRect`), which takes into account the scaling. We want the absolute value without scaling, so we're better off directly using the DOM properties.